styles/style.css

```diff
@@ -95,7 +117,38 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+
+.buttonBackground {
```

Comment:
Extra tip:
When a mouse goes on any clickable item (such as button), we want to tell users to know that's clickable. For that, we convert the cursor into a hand that points to, similar to the link you have. You might want to google `css cursor` to learn more.